actionmailer/lib/action_mailer/preview.rb

```diff
@@ -25,7 +25,31 @@ module Previews #:nodoc:
       mattr_accessor :preview_interceptors, instance_writer: false, default: [ActionMailer::InlinePreviewInterceptor]
     end
 
+    def preview_path
+      ActiveSupport::Deprecation.warn(<<-MSG.squish)
+        Using preview_path option is deprecated and will be removed in Rails 6.0.
+        Please use preview_paths instead.
+      MSG
+      self.class.preview_paths.first
+    end
```

Comment:
This instance method is currently private, so I think it can be removed without deprecation.

Reply:
Looks like instance reader is public https://github.com/rails/rails/blob/bce675eac49a8673d63ea41991f020aa3aa8c506/actionmailer/lib/action_mailer/preview.rb#L14

Reply:
You‘re right, sorry. 😅